tools/build-config

```diff
@@ -22,15 +18,23 @@ const createXcconfigs = async (buildConfigName, directory) => {
 }
 
 program
-  .command('config <build_config_name>')
+  .command('to-xcconfig <build_config_name>')
+  .requiredOption('--directory <directory>', 'the directory to put the created xcconfig file in')
   .description('create and write a new .xcconfig from the iosBuildOptions of the specified build config')
-  .action(async buildConfigName => {
+  .action(async (buildConfigName, cmdObj) => {
```

Comment:
What is `cmdObj` and what is it used for?

Reply:
see [here](https://github.com/tj/commander.js/#action-handler-subcommands) . And I guess it is used do get the directory where the initial command was initiated from in the command line

Reply:
Yes this is the API of commander.js